core/ui_mixins/quick_panel.py

```diff
@@ -500,3 +500,30 @@ def on_selection(self, commit):
     def on_selection_async(self, commit):
         sublime.active_window().run_command("hide_panel", {"panel": "output.show_commit_info"})
         self.on_done(commit)
+
+
+def show_stash_panel(entries, on_done):
```

Comment:
Then the entries should be generated in the `show` method of `StashPanel` as in the [`show_remote_panel`](https://github.com/divmain/GitSavvy/blob/cf2b6e3992a93b5bbae1d3d8b963d327949a1ea6/core/ui_mixins/quick_panel.py#L140)